### DISCUSSION TITLE
spacecount vs identation.

### DISCUSSION THREAD

**Discussion by rasjani** [MarketSquare/robotframework-tidy#700] [General]
This is just matter of personal taste so feel free to ignore if this is of no concert to anybody.

I personally prefer to use 2 spaces as indentation and 3 for for separating arguments.

However, on tidy configs, spacecount affects both, indentation **and** argument separating.   Would it be possible to have 2 config options for both of these ?

**Comment by bhirsz** [comment]
We have separate setting for indentation and separator between tokens. By default if not configured, it will be the same (4 spaces). See those settings:

![image](https://github.com/MarketSquare/robotframework-tidy/assets/8532066/a41f789b-faeb-443b-a566-9acfb1c817a0)

And if it's not enough (ie you want different number of spaces depending on type of argument), I have opened this issue https://github.com/MarketSquare/robotframework-tidy/issues/637
